src/main/java/ratpack/zipkin/TracedParallelBatch.java

```diff
@@ -27,8 +27,12 @@
  * which do not "inherit" registries from the calling execution. Consequently,
  * the trace context must be passed explicitly to the forked executions.
  *
+ * @deprecated As of 2.4 there is now a {@link ratpack.exec.ExecInitializer} provided by {@link ServerTracingModule}
```

Comment:
The version is 2.3.2 now right?

Reply:
Ha yeah right.  I was thinking Ratpack version here.

Since this wasn't available in older versions of Ratpack, maybe it would help to include the Ratpack version as well.

As a suggestion:
`@deprecated As of 2.4 since Ratpack 1.4 and later has {@link ratpack.exec.ExecInitializer} which we register an instance of through {@link ServerTracingModule}`?  Just a suggestion for folks that read to quickly ;)